### ISSUE TITLE
In cluster-audit.sh output, provide text to indicate problems.

### ISSUE THREAD

**Issue by dweelea** [jbenninghoff/cluster-validation#36]
The output produced provides information on the status of certain things, without any indication of whether it’s a problem.  For example, on my cluster audit, I got this message, which is good (as it should be):
 
SElinux status: SELINUX=disabled
Disabled

And this message, which is bad (not as it should be):
 
Required RPMs:
package ntp is not installed
 
And this message, which is neutral:
 
mapr account for MapR Hadoop
mapr user NOT found!
 
(and the use of NOT in all caps could be taken to indicate this is a problem).
 
It would be nice if a setting that was problematic had a WARN: (or something) in front of it, so people could look for that keyword and make sure they’re not missing problems (or interpreting something as problematic when it’s not).

**Comment by jbenninghoff** [comment]
True.  Good feedback.  Will take some effort to address all output.
